### DISCUSSION TITLE
prettier throws a lot of mistakes

### DISCUSSION THREAD

**Discussion by tinohager** [LycheeOrg/Lychee#2752] [General]
`npm run check-formatting`

[warn] resources/js/views/Settings.vue
[warn] resources/js/views/Sharing.vue
[warn] resources/js/views/Statistics.vue
[warn] resources/js/views/Users.vue
[warn] resources/js/vite-env.d.ts
[warn] resources/sass/app.scss
[warn] resources/sass/fonts.css
[warn] Code style issues found in 176 files. Run Prettier with --write to fix.

**Comment by ildyria** [comment]
that's weird.
```
▶ npm run check-formatting

> check-formatting
> prettier --check resources/js/ resources/sass/

Checking formatting...
All matched files use Prettier code style!
```

I do not have the same.

**Reply by ildyria** [reply]
`npm run format` should fix the formatting for you though.

**Reply by tinohager** [reply]
![image](https://github.com/user-attachments/assets/b0acd438-9a87-4927-adf8-09595a17271e)

**Reply by ildyria** [reply]
That is very very weird. It feels like it is ignoring the configuration.

**Reply by tinohager** [reply]
something with CR LF / LF ...

**Reply by ildyria** [reply]
should be fixed with : #2875

**Reply by ildyria** [reply]
Should be fixed with latest commit, though it will require to re clone the project.
